DESCRIPTION

```diff
@@ -34,6 +34,7 @@ Depends: R (>= 3.5)
 Imports:
     cli,
     dplyr (>= 1.0.5),
+    glue,
```

Comment:
If lifecycle has this as an Import do we need to have it as an import

Reply:
if we need it can we set the minimum version like the rest of the imports

Reply:
min version added! yep, if we use the package top-level, it needs to be added here. it's a dependency of lifecycle, dpylr, stringr, and a bunch of posit pkgs (and others)